Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1140,5 +1140,6 @@
     <string name="test_api_body_other_keys">"Other key"</string>
     <!-- NOTR -->
     <string name="test_api_calculate_risk_level">"Calculate Risk Level"</string>
+    <string name="test_api_country_filter_status">Country filter applied for: \n%1$s</string>
```

Comment:
Unlocalized?

Reply:
Already discussed with @harambasicluka and fixed